ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/MacOSX.yml

```diff
@@ -16,13 +15,18 @@ Build_Tool_Packages:
   - nasm # openj9 jdk13+
   - wget
 
+Build_Tool_Packages_NOT_arm64:
+  - bash # OpenJ9 needs bash v4 or later
```

Comment:
Does Big Sur suddenly contain a modern bash?

Reply:
currently, Big Sur only support Bash v3. We'll have to wait until it's supported on arm64

Reply:
So we need a comment that explains that: "OpenJ9 needs Bash 4 or newer but it is not available on arm64, yet." or something like that. And we probably need an issue here that reminds us to fix this as soon as Bash becomes available.

Sorry for obsessing over stuff like that but I had to do too much repository archaeology recently ☹️

Reply:
Does this say we're NOT installing any bash on the arm64 installs?

Reply:
no it comes with bash 3 by default, it's just that the bash that homebrew tries to install is not supported